main.go

```diff
@@ -119,8 +118,15 @@ func main() {
 func createTemplates(envVars server.EnvironmentVars) map[string]*template.Template {
 	templates := map[string]*template.Template{}
 	templateFunctions := map[string]interface{}{
-		"join": func(sep string, items []string) string {
-			return strings.Join(items, sep)
+		"showFinance": func(roles []string) bool {
```

Comment:
I'm wondering if it would be better to pass the list of roles to the header component and move these visibility rules to the header repo? Otherwise we'd have to re-implement these rules in every single service 🤔